tests/schema/test_validate_schema.py

```diff
@@ -0,0 +1,86 @@
+    "sns_outside_sqs",  # 8 is not of type string
+    "function_with_cwe_dlq_and_retry_policy",  # Doesn't match schema at all...
+    "function_with_cwe_dlq_generated",  # Doesn't match schema at all...
+    "function_with_request_parameters",  # RequestParameters don't match documentation. Documentation and its example don't match either
+    "api_with_request_parameters_openapi",  # RequestParameters don't match documentation. Documentation and its example don't match either
+    "api_with_aws_iam_auth_overrides",  # null for invokeRole
+    "eventbridgerule",  # missing required field 'Patterns'
+    "self_managed_kafka_with_intrinsics",  # 'EnableValue' is of type bool but defined as string
+    "api_with_resource_policy_global",  # 'ResourcePolicy CustomStatements' output expects a List
+    "api_with_resource_policy",  # 'ResourcePolicy CustomStatements' output expects a List
+    "api_with_if_conditional_with_resource_policy",  # 'ResourcePolicy CustomStatements' output expects a List
+    "api_rest_paths_with_if_condition_swagger",  # 'EnableSimpleResponses' and 'AuthorizerPayloadFormatVersion' not defined in documentation
+    "api_rest_paths_with_if_condition_openapi",  # 'EnableSimpleResponses' and 'AuthorizerPayloadFormatVersion' not defined in documentation
+    "state_machine_with_api_authorizer_maximum",  # 'UserPoolArn' expects to be a string, but received list
+    "api_with_auth_all_maximum",  # 'UserPoolArn' expects to be a string, but received list
+    "api_with_auth_and_conditions_all_max",  # 'UserPoolArn' expects to be a string, but received list
+    "api_with_auth_all_maximum_openapi_3",  # 'UserPoolArn' expects to be a string, but received list
+    "api_with_authorizers_max_openapi",  # 'UserPoolArn' expects to be a string, but received list
+    "api_with_authorizers_max",  # 'UserPoolArn' expects to be a string, but received list
+    "api_with_any_method_in_swagger",  # Missing required field 'FunctionArn'
+    "api_with_cors_and_only_headers",  # 'AllowOrigins' is required field
+    "api_with_cors_and_only_methods",  # 'AllowOrigins' is required field
+    "implicit_api_with_auth_and_conditions_max",  # 'UserPoolArn' expects to be a string, but received list
+]
+
+
+def should_skip_test(s: str) -> bool:
+    for test in SKIPPED_TESTS:
+        if test in s:
+            return True
+    return False
+
+
+def get_all_test_templates():
+    return (
+        list(Path("tests/translator/input").glob("**/*.yaml"))
+        + list(Path("tests/translator/input").glob("**/*.yml"))
+        + list(Path("tests/validator/input").glob("**/*.yaml"))
+        + list(Path("tests/validator/input").glob("**/*.yml"))
+        + list(Path("integration/resources/templates").glob("**/*.yaml"))
+        + list(Path("integration/resources/templates").glob("**/*.yml"))
+    )
+
+
+SCHEMA_VALIDATION_TESTS = [os.path.splitext(f)[0] for f in get_all_test_templates() if not should_skip_test(str(f))]
```

Comment:
Why the split?

Reply:
This is for showing the exact template name when running pytest. Otherwise, it will only have numerics.
<img width="1275" alt="Screen Shot 2022-11-22 at 1 00 17 PM" src="https://user-images.githubusercontent.com/28763956/203431410-c740cb24-5c0a-43f9-a0b3-ff7604e86401.png">